helix-term/tests/test/helpers.rs

```diff
@@ -322,6 +322,12 @@ impl AppBuilder {
     }
 }
 
+pub async fn run_event_loop_to_idle(app: &mut Application) {
```

Comment:
```suggestion
pub async fn run_event_loop_until_idle(app: &mut Application) {
```
matching `app.event_loop_until_idle(...)`